backend/infrahub/core/diff/exceptions.py

```diff
@@ -0,0 +1,12 @@
+from neo4j.graph import Path
```

Comment:
Potentially add a `from __future__ import annotations` and move this import into a TYPECHECKING block.

Not sure where it will be imported but it's probably better to be sure if we later run agents that doesn't have access to the database then we don't want the neo4j module to be loaded at all.

Reply:
good idea. made some updates to do this